src/OSPSuite.Core/Domain/Services/SpatialStructureMerger.cs

```diff
@@ -89,6 +89,18 @@ private void createMergedContainerStructureInRoot(IContainer root, SimulationBui
          }
       }
 
+      private void tryMergeTopContainerInStructure(IContainer root, IContainer topContainer, MergeBehavior mergeBehavior, (SpatialStructure spatialStructure, MergeBehavior mergeBehavior) item)
```

Comment:
then we wouldn't need the Tuple in here, and instead we'd only have 

`private void tryMergeTopContainerInStructure(IContainer root, IContainer topContainer, MergeBehavior mergeBehavior, SpatialStructure spatialStructure)`